api/index.ts

```diff
@@ -66,6 +68,29 @@ const start = async () => {
     })
   });
 
+  // Swagger plugin
+  if (NODE_ENV === 'development') {
```

Comment:
```suggestion
  if (FCC_NODE_ENV === 'development') {
```

Reply:
The environment value is actually `NODE_ENV`

Ref - https://github.com/freeCodeCamp/freeCodeCamp/blob/main/api/utils/env.ts#L42

Reply:
You missed this in the meeting. Mrugesh has suggested (authoritatively) we not personally use the `NODE_ENV` variable. In this specific case, he just wants us to rename that variable so the two are not confused.

Reply:
_Your wish is my command_

I'll update the variable across all the files later today

Reply:
Let's use `FREECODECAMP_NODE_ENV`, for continuity with the old api. Otherwise it's another thing we have to update for no obvious gain.

Reply:
I've replaced it with `FCC_ENABLE_SWAGGER_UI`